### ISSUE TITLE
Create New Ticket in UvDesk

### ISSUE THREAD

**Issue by sbonzanni** [uvdesk/core-framework#199]
When I create a new ticket, I generate a new Customer. Can this be avoided?

**Comment by papnoisanjeev** [comment]
Hello sbonzanni,

In UVDesk we have Ticketing system in which we need to resolve customer query and concern.
If someone is creating a ticket with a new email then he will be treated as a customer and his account will be created.(only once with a new email Id) as it is a basic procedure. we need to reply to the customer on the ticket so we need the information of the customer.

For any query or doubts you can raise a support ticket on **support@uvdesk.com**
